white_paper/clause_2.adoc

```diff
@@ -4,26 +4,26 @@ This section describes the beneficiaries & benefits of representing data, includ
 
 === Beneficiaries
 
-==== Beneficiary 1: Data Consumers
+==== Beneficiary 1: Data consumers
 
-Data consumers use data, including geospatial data. They key needs are:
+Data consumers use data, including geospatial data. Their key needs are:
 
-1. to access and understand data from multiple sources; and
-2. to integrate data - geospatial and non-geospatial
-3. to get a data quality evaluation of the given geospatial data
+1. to access and understand data from multiple sources,
+2. to integrate data - geospatial and non-geospatial, and
+3. to get a data quality evaluation of the given geospatial data.
 
-==== Beneficiary 2: Data Publishers
+==== Beneficiary 2: Data publishers
 
 Data publishers have data, including geospatial data. Their key needs are:
 
-1. to maximise use of their data; and
-2. to manage their data as simply as possible
-3. to indicate possible usage contexts of the geospatial data
-4. to quality-assure their geospatial data
+1. to maximise use of their data,
+2. to manage their data as simply as possible,
+3. to indicate possible usage contexts of the geospatial data, and
```

Comment:
```suggestion
3. to indicate possible usage contexts of the geospatial data; and
```

Reply:
I wondered about conventions for punctuation for list in English and the first hit I got on Google showed the method I uses. But this page seems authoritative enough, and it demonstrates using semicolons: http://www.plainenglish.co.uk/punctuating-bulleted-lists.html. So yeah, that seems right.